installer/pkg/cmd/gcp_broker.go

```diff
@@ -236,17 +268,38 @@ func getOrCreateVirtualBroker(projectID, brokerName, brokerTitle string) (*virtu
 		return nil, fmt.Errorf("failed to create broker client. You might want to run 'gcloud auth application-default login'")
 	}
 
-	brokerURL := "https://servicebroker.googleapis.com"
+	host := "https://servicebroker.googleapis.com"
+	brokerURL := fmt.Sprintf("%s/v1beta1/projects/%s/brokers/%s", host, projectID, brokerName)
 	errCode, respBody, err := brokerClient.CreateBroker(&adapter.CreateBrokerParams{
-		URL:     brokerURL,
+		URL:     host,
 		Project: projectID,
 		Name:    brokerName,
 		Title:   brokerTitle,
 	})
-	if errCode == 409 {
+	if errCode == http.StatusConflict {
+		fmt.Printf("Broker %q, already exists\n", brokerName)
+		res, err := brokerClient.GetCatalog(&adapter.GetCatalogParams{
+			Server: brokerURL,
+		})
+		if err != nil {
+			return nil, fmt.Errorf("Invalid broker %q, error getting catalog for broker: %v\n", brokerURL, err)
+		}
+
+		isEAPBroker := false
+		for _, svc := range res.Services {
+			if _, ok := eapServiceGuids[svc.ID]; ok {
+				isEAPBroker = true
+				break
+			}
+		}
+
+		if isEAPBroker {
+			return nil, fmt.Errorf("Existing broker is an EAP broker, please delete broker using broker-cli, and rerun \"sc add-gcp-broker\" to get Beta broker\n")
```

Comment:
Your existing broker is deprecated. Please delete your broker using broker-cli and re-run sc add-gcp-broker.

Reply:
done